### ISSUE TITLE
Mild oligozoospermia

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9798]
**Preferred term label:**
Mild oligozoospermia

**Synonyms**

**Definition (free text, please give PubMed ID)**
>30 - <39 million sperm (>10 - <15 million/mL concentration)

**Parent term (use hpo.jax.org/app)**
HP:0000798

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399805

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr 

added with definition
A type of oligozoospermia characterized by a total number of more than 30 but less than 39 million sperm in the ejaculate or a concentration of more than 10 but less than 15 million sperm per milliliter.

HP:0034816
